tests/ApiApprovalTests/ApiApprovalTests.csproj

```diff
@@ -11,7 +11,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Authorization.AspNetCore\Authorization.AspNetCore.csproj" />
```

Comment:
I really have no idea how the last set of build checks passed in #803 without this being removed.  (!!!)